package.json

```diff
@@ -5,6 +5,9 @@
   "repository": "git@github.com:Boaterfly/async-responsive-iframe.git",
   "author": "nfabredev <nicolas.fabre@boaterfly.com>",
   "license": "MIT",
+  "engines": {
+    "node": "8.11.1"
```

Comment:
IMO this is way too restrictive.
And I would go for the minimal node version that is compatible and upwards.

Probably something like this
```json
"engines": {
    "node": ">=6.0.0"
  },
```
(see [what `lodash` is doing](https://github.com/lodash/lodash/blob/fe7da419562108b1da4226cf9107f63d697996b1/package.json#L7-L9))

Reply:
Yeah sure, i put `6.4.0`, check the commit message